### PR TITLE
Added end block time

### PR DESCRIPTION
**Pull request by Natnael-kibe**
Now we have a box that says end time.

### CODE REVIEW

**Comment by Natnael-kibe** [comment]
Codacy will yell at this. Please merge anyway. Thanks.

File: index.html [hunk 1]
```diff
@@ -73,12 +74,12 @@ <h3 class="panel-title">Day</h3>
         </div>
       </div>
 
-      <div class="panel panel-danger" id="block-panel">
+      <div class="panel panel-danger">
         <div class="panel-heading">
-          <h3 class="panel-title">Current Block</h3>
+          <h3 class="panel-title" id="Derrick">Until the End of Block</h3>
```
Comment (HeroCC) [inline index.html]: Descriptive names, please

Comment (HeroCC) [reply index.html]: Also, 'Until the End of Block' is a bit long and wraps on mobile screens, can you make it a few less words?

File: index.html [hunk 2]
```diff
@@ -73,12 +74,12 @@ <h3 class="panel-title">Day</h3>
...
         </div>
         <div class="panel-body">
-          <span id='blockId'>Z</span> Block
+          <span id='blockTime'>Test...</span>
```
Comment (HeroCC) [inline index.html]: No! Descriptive errors, like 'Calculating' or 'Loading' or even 'Unknown' or '???'

File: js/Timer.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    ["Advisory", "A Block", "B Block", "C Block", "First Lunch (E Block)", "Second Lunch (E Block)", "Third Lunch (E Block)", "D Block", "G Block"],
+    ["Advisory", "A Block", "C Block", "D Block", "First Lunch (G Block)", "Second Lunch (G Block)", "Third Lunch (G Block)", "E Block", "F Block"],
+    ["Advisory", "A Block", "B Block", "C Block", "First Lunch (F Block)", "Second Lunch (F Block)", "Third Lunch (F Block)", "G Block", "D Block"],
+    ["Advisory", "A Block", "C Block", "B Block", "First Lunch (G Block)", "Second Lunch (G Block)", "Third Lunch (G Block)", "F Block", "E Block"],
+    ["Advisory", "B Block", "C Block", "D Block", "First Lunch (E Block)", "Second Lunch (E Block)", "Third Lunch (E Block)", "F Block", "G Block"],
+  ];
+
+  document.getElementById("blockTime").innerHTML = Lhours + "h " + Lminutes + "m " + Lseconds + "s ";
+
+  if (dy ===0){
```
Comment (HeroCC) [inline js/Timer.js]: Add a space here

File: js/Timer.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    document.getElementById("blockTime").innerHTML = "Timer not available.";
```
Comment (HeroCC) [inline js/Timer.js]: Get rid of the period and do a descriptive message with most likely error, like 'No School'. Or a countdown until Monday?

File: js/main.js [hunk 5]
```diff
@@ -99,6 +102,7 @@ function main(){
       var block = (aspenInfo.schedule.block);
       var blockSchedule = (aspenInfo.schedule.blockSchedule);
       var day = (aspenInfo.schedule.day);
```
Comment (HeroCC) [inline js/main.js]: Instead of making another variable 'dy', just make 'day' declared in the global scope, and then set it on this line.

**Review by HeroCC — CHANGES_REQUESTED** [review]
Overall good, but a few QOL changes. Thanks!

**Review by HeroCC — APPROVED** [review]

**Pull request merged**